이건우/week7/leetcode1480.py

```diff
@@ -0,0 +1,12 @@
+class Solution(object):
+    def runningSum(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: List[int]
+        """
+        res = []
+        sums = 0
+        for _ in nums:
+            sums += _
+            res.append(sums)
+        return res
```

Comment:
아니면 꼼수로 Python의 itertools 라이브러리의 accumulate를 사용해서 바로 돌려주는 방법도 있을 것 같네요!
```suggestion
from itertools import accumulate

class Solution(object):
    def runningSum(self, nums):
        return list(accumulate(nums))
```